www/polj10/sp1/nastroje/transcriptorium.html

```diff
@@ -70,9 +71,9 @@
         <main>
             <!-- Zdroj - http://vokabular.ujc.cas.cz/moduly/nastroje/transcriptorium/napoveda -->
             <div class="poster">
-                <ul class="breadcrumbs">
+                <ul class="breadcrumbs print-hide">
```

Comment:
kolik krat jsi musel dopsat print-hide? nestacilo vzit jiz deklarovane tridy a dat je to media query?

Reply:
šlo, ALE co kdybych z nějakého důvodu chtěl v budoucnu mít některé breadcrumbs součástí tisku?
Aby bylo poznat, že některá stránka patří k tomu, či onomu projektu?
To je ale spíš "filosofická" otázka :)